packages/react-scripts/config/loaders/getJSLoaders.js

```diff
@@ -0,0 +1,103 @@
+'use strict';
```

Comment:
This code is identical in two configs. Extracting helps to avoid undesired mismatch